deploy/cloudbuild-release.yaml

```diff
@@ -16,7 +16,7 @@ steps:
            "-t", "gcr.io/kaniko-project/executor:debug-$TAG_NAME", "."]
   - name: "gcr.io/cloud-builders/docker"
     args: ["tag", "gcr.io/kaniko-project/executor:debug-$TAG_NAME", 
-           "gcr.io/kaniko-project/executor:debug"]
```

Comment:
We would also like to keep the existing `debug` tag since a lot of folks depend on this tag

Reply:
Okay